client/my-sites/importer/importer-wordpress.jsx

```diff
@@ -44,9 +44,10 @@ class ImporterWordPress extends React.PureComponent {
 
 		importerData.uploadDescription = this.props.translate(
 			'Upload a {{b}}WordPress export file{{/b}} to start ' +
-				'importing into {{b2}}%(title)s{{/b2}}. Check out our ' +
-				'{{inlineSupportLink/}} if you need ' +
-				'help exporting the file.',
+				'importing into {{b2}}%(title)s{{/b2}}. A WordPress export is ' +
+				'an XML file with your page and post content, or a zip archive ' +
```

Comment:
I may be wrong, but this comma doesn't seem correct. The last bit (`or a zip archive`) isn't an independent clause. 

https://www.grammarly.com/blog/comma-before-or/

Reply:
Hm, it reads OK to me Jeff. The extra conjunction in "page and post content" adds a bit of complexity to the sentence – I wouldn't say this is the "two short grammatical elements" Grammarly's talking about. The comma breaks the chain of words up and makes it clear that "containing several XML files" belongs with "zip archive". Here it is without the comma:

> A WordPress export is an XML file with your page and post content or a zip archive containing several XML files.

IMHO some people might need to read that twice to get the meaning. Then again that's just me – if other people also find it looks odd we could tweak it when we change the text of the link.

Reply:
I agree. Without the comma, the sentence is hard to understand.